cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,22 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io');
```

Comment:
Basically, it is good that you know it and you use it. But in this task, where you have only one "it" it is not needed.